src/platform/bizhawk/bizinterface.c

```diff
@@ -0,0 +1,623 @@
+#include <stdlib.h>
+#include <stdint.h>
+#include "mgba/core/core.h"
+#include "mgba/core/log.h"
+#include "mgba/core/timing.h"
+#include "mgba/core/serialize.h"
+#include "mgba/core/blip_buf.h"

```

Comment:
blip_buf will need to be replaced with mAudioBuffer and mAudioResampler to work anymore.